src/Build/Definition/Project.cs

```diff
@@ -3733,7 +3741,8 @@ private void ReevaluateIfNecessary(ILoggingService loggingServiceForEvaluation,
                     s_buildEventContext,
                     evaluationContext.SdkResolverService,
                     BuildEventContext.InvalidSubmissionId,
-                    evaluationContext);
+                    evaluationContext,
+                    _interactive);
```

Comment:
Its difficult to tell in this diff but this is where the `interactive` flag is passed to the evaluator.